Gopkg.lock

```diff
@@ -718,6 +718,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "74819346c715a1b94c8c7c68814578a73bc05d29cc79e35234e99a65d46d8b20"
+  inputs-digest = "6297d67389d3671b266052a872fa7588d14c78fff90b648fcd191beb3159d7df"
```

Comment:
How did this happen?

Reply:
either upstream BUILD file change got negated by gazelle or some symlinks in the repo changed (which are removed after `dep ensure` in `tools/dep_ensure.sh`)